packages/cli-kit/src/node/hooks/prerun.ts

```diff
@@ -8,5 +9,5 @@ export const hook: Hook.Prerun = async (options) => {
   const command = cmd.replace(/:/g, ' ')
   const args = options.argv
   debug(`Running command ${command}`)
-  await start({command, args, commandClass: options.Command})
+  await start({command, args, commandClass: options.Command as unknown as typeof Command})
```

Comment:
@shauns I'm assuming here that all plugins use the base command class exported by `cli-kit`. I think it's a safe assumption but want to know your thoughts about this.